examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/supervisorsManager.py

```diff
@@ -0,0 +1,46 @@
+from Args import ModelArgs
+import PPORun
+
+"""
+Modify these constants if needed.
+"""
+# The project name on the wandb
+PROJECT_NAME = "Test"
+WANDB_SAVE_FREQ = 100  # Logging Frequency on wandb
```

Comment:
Logging frequency in steps or? Can we make it more clear in the comment?
```suggestion
wandb_save_frequency = 100  # Logging frequency on wandb
```

Reply:
@tsampazk I think renaming it to `wandb_save_period` is better.

Reply:
Yes @KelvinYang0320  sounds good, maybe specify in the comment that it is counted in steps?

Reply:
Yeah, I have done this. Is [this line](https://github.com/KelvinYang0320/deepworlds/blob/e99bc32d75ae8de4b4816a4146c667757558ecb1/examples/khr-3hv/khr-3hv_continuous/controllers/robot_supervisor_manager/robot_supervisor_manager.py#L8) okay for you?

Reply:
Yes that's great thanks. :smile: